### PR TITLE
Add tagto configs

### PR DESCRIPTION
**Pull request by xiliu8006**
Add Tag(DATASET, ALGORITHM, BACKBONE, OTHERS) to configs' README.md

### CODE REVIEW

File: configs/fp16/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Introduction
 
+[ALGORITHM]
```
Comment (ZwwWayne) [inline configs/fp16/README.md:5]: OTHERS

File: configs/nuimages/README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## Introduction
 
+[ALGORITHM]
```
Comment (ZwwWayne) [inline configs/nuimages/README.md:5]: DATASET

File: configs/ssn/README.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 ## Introduction
 
+[BACKBONE]
```
Comment (ZwwWayne) [inline configs/ssn/README.md:5]: ALGORITHM

**Review by ZwwWayne — APPROVED** [review]

**Comment by ZwwWayne** [comment]
FYI @AemikaChow

**Comment by OpenMMLab-Coodinator** [comment]
Hi @xiliu8006! First of all, we want to express our gratitude for your significant PR in this project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**